ui/page/dcrdex/market.go

```diff
@@ -1796,14 +1812,13 @@ func (pg *DEXMarketPage) refreshOrders() {
 			for _, ord := range mkt.InFlightOrders {
 				pg.orders = append(pg.orders, &clickableOrder{Order: ord.Order})
 			}
-
-			// We've just appended new orders, sort to ensure newest orders are
-			// displayed first.
-			sort.SliceStable(orders, func(i, j int) bool {
-				return pg.orders[i].SubmitTime > orders[j].SubmitTime
-			})
 		}
 	}
+
+	// Always sort orders.
+	sort.SliceStable(pg.orders, func(i, j int) bool {
+		return pg.orders[i].SubmitTime > orders[j].SubmitTime
```

Comment:
Had a out-of-range panic here once.
```suggestion
		return pg.orders[i].SubmitTime > pg.orders[j].SubmitTime
```